migtests/tests/mysql-tests/mysql-datatypes/validate

```diff
@@ -52,6 +52,19 @@ EXPECTED_SUM_OF_COLUMN = {
 
 EXPECTED_ENUM_VALUES = ['low', 'high', 'med']
 
+EXPECTED_DATA_TYPES = {
+	'floating_point_types': {'double_type': 'double precision', 'real_type': 'double precision', 'id': 'bigint', 'float_type': 'double precision'},
```

Comment:
Change the layout of this dict for better readability:
```
EXPECTED_DATA_TYPES = {
	'floating_point_types': {
        'double_type': 'double precision',
		'real_type': 'double precision',
		'id': 'bigint',
		'float_type': 'double precision'
	},
```

Reply:
Done